Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridComboBoxColumn.cs

```diff
@@ -0,0 +1,606 @@
+            }
+        }
+
+        private void RefreshForeground(TextBlock textBlock, Brush computedRowForeground)
+        {
+            if (this.Foreground == null)
+            {
+                if (computedRowForeground != null)
+                {
+                    textBlock.Foreground = computedRowForeground;
+                }
+            }
+            else
+            {
+                textBlock.Foreground = this.Foreground;
+            }
+        }
+
+        private void SetTextFontSize(DependencyObject textElement, DependencyProperty fontSizeProperty)
+        {
+            double newFontSize = this.FontSize;
+            if (double.IsNaN(newFontSize))
+            {
+                textElement.ClearValue(fontSizeProperty);
+            }
+            else
+            {
+                textElement.SetValue(fontSizeProperty, newFontSize);
+            }
+        }
+
+        private bool EnsureOwningGrid()
+        {
+            if (this.OwningGrid != null)
+            {
+                if (this.OwningGrid != _owningGrid)
+                {
+                    _owningGrid = this.OwningGrid;
+                    _owningGrid.LoadingRow += OwningGrid_LoadingRow;
+                    _owningGrid.CellEditEnded += OwningGrid_CellEditEnded;
+                }
+
+                return true;
+            }
+
+            return false;
+        }
+
+        private void OwningGrid_LoadingRow(object sender, DataGridRowEventArgs e)
+        {
+            SetDisplayMemberPathValue(e.Row);
+        }
+
+        private void OwningGrid_CellEditEnded(object sender, DataGridCellEditEndedEventArgs e)
+        {
+            SetDisplayMemberPathValue(e.Row);
+        }
+
+        private void SetDisplayMemberPathValue(DataGridRow row)
+        {
+            if (this.OwningGrid != null)
+            {
+                var textBlock = this.GetCellContent(row) as TextBlock;
+                if (textBlock != null && DisplayMemberPath != null)
+                {
+                    var displayValue = GetDisplayValue(row.DataContext);
+
+                    textBlock.Text = displayValue;
+                }
+            }
+        }
+
+        private string GetDisplayValue(object dataItem)
+        {
+            if (this.Binding != null && dataItem != null)
+            {
+                var value = dataItem.GetType().GetProperty(Binding?.Path.Path).GetValue(dataItem);
+
+                var items = ItemsSource as IEnumerable<object>;
+
+                var item = items?.FirstOrDefault(x => x.GetType().GetProperty(Binding.Path.Path).GetValue(x).Equals(value));
+
+                var displayValue = item?.GetType().GetProperty(DisplayMemberPath).GetValue(item) ?? string.Empty;
+
+                return displayValue as string ?? displayValue.ToString();
+            }
+
+            return string.Empty;
+        }
+
+        private void EnsureColumnBinding(Binding binding, object dataItem)
+        {
+            if (binding == null)
```

Comment:
You're always passing in this.Binding to this method, and then use a mix of 'binding' and 'Binding' in it.
I suggest you remove the 'binding' argument, and use this.Binding everywhere in the method.